daemon/daemon_unix.go

```diff
@@ -419,6 +419,7 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		netlabel.DriverMTU:        strconv.Itoa(config.Mtu),
 		bridge.EnableIPMasquerade: strconv.FormatBool(config.Bridge.EnableIPMasq),
 		bridge.EnableICC:          strconv.FormatBool(config.Bridge.InterContainerCommunication),
+		netlabel.EnableIPv6:       strconv.FormatBool(config.Bridge.EnableIPv6),
```

Comment:
`netOption` is really a collection of per network specific options. We consider  `EnableIPv6` a driver specific option instead, which is why it is being passed down separately from the network options.